### ISSUE TITLE
Ticket Builder: User can add ticket items from an expired pricelist in Console

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#1184]
**[Mingle Card: 1367](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/1367)**
**2014/01/21 - test again once a new managed package is ready.**

QA note:
--------

Created Price Book in Console with expiration date = 1/19/2014; Jobs>Add new job> Create new Ticket for that job>add PB items:

Items from expired PB available for selection

(Example : Console login/pw : [np\_console@test2.com/computer#4](mailto:np_console@test2.com/computer#4), PB "3apa3a", job "ConsoleJ\_ExpiredPB" , ticket T-8AC-16D-048E)

DEV Notes:
----------

The "Pricebook flyout" is now hidden if the pricebook is expired.

If the pricebook is expired there will now be a message displayed on the page.

Make sure to test the edge cases around todays date - the pricebook should expire at midnight of the day set.

Steps to reproduce
------------------

**OS: Windows**

**Browser: Chrome**

**Username/Password: ajfield@qa.com/computer#4**

1.  Set the pricebook expiry date before today's date
2.  Create a Job with that PB
3.  Create a ticket on that Job and try to add some ticket items
    1.  Please note, this is only a defect if it is in Ticket Builder. This is not a defect if it happens in the declarative screen

Expected result
---------------

Should not allow user to add any ticket items from the expired price book

Actual Result
-------------

Allows it

Related Cards
-------------

**Comment by githubbob42** [comment]
### Related Story: #598 Restrict Pricelists by Effective and Expiration Date ([Mingle](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/748))
